js/current-selection/index.js

```diff
@@ -2,14 +2,26 @@ import wp from 'wp';
 
 import CurrentSelection from './components/current-selection';
 
-const { withSelect } = wp.data;
+const { withSelect, dispatch } = wp.data;
+
+dispatch( 'core' ).addEntities( [
+	{
+		name: 'post-select',
+		kind: 'hm-gb-tools/v1',
+		baseURL: '/hm-gb-tools/v1/post-select',
+	},
+] );
 
 export default withSelect( ( select, ownProps ) => {
 	const { getEntityRecords } = select( 'core' );
 	const { postIds, postType } = ownProps;
-	const posts = getEntityRecords( 'postType', postType, {
+	const postTypes = Array.isArray( postType ) ? postType : [ postType ];
+	const posts = getEntityRecords( 'hm-gb-tools/v1', 'post-select', {
 		include: postIds,
+		per_page: postIds.length,
```

Comment:
This doesn't really benefit us given the `include`, though I grant it does not hurt